enterprise/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/compatibility/v3_3/runtime/RegisteredRewriter.scala

```diff
@@ -126,6 +136,18 @@ class RegisteredRewriter(tokenContext: TokenContext) {
           case _ =>
             throw new InternalException("Did not find `" + k + "` in the pipeline information")
         }
+
+      case idFunction@FunctionInvocation(_, FunctionName("id"), _, _) =>
+        idFunction
+
+      case _: FunctionInvocation =>
```

Comment:
As far as I know, most functions do not read or write to execution contexts, and should be fine to run in the interpreted runtime